authentik/core/api/authenticated_sessions.py

```diff
@@ -115,3 +123,14 @@ class AuthenticatedSessionViewSet(
     filterset_fields = ["user__username", "session__last_ip", "session__last_user_agent"]
     ordering = ["user__username"]
     owner_field = "user"
+
+    @action(detail=False, methods=["POST"], pagination_class=None, filter_backends=[])
+    def bulk_delete(self, request: Request) -> Response:
+        """Bulk revoke all sessions for multiple users"""
+        serializer = BulkDeleteSessionSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
```

Comment:
```suggestion
    @validate(BulkDeleteSessionSerializer)
    def bulk_delete(self, request: Request, body: BulkDeleteSessionSerializer) -> Response:
        """Bulk revoke all sessions for multiple users"""
```
You can then use `body.validated_data`.
The import is `from authentik.api.validation import validate`

Reply:
Also, if we change this to DELETE, the following would be correct:
```suggestion
    @validate(BulkDeleteSessionSerializer, location="query")
    def bulk_delete(self, request: Request, query: BulkDeleteSessionSerializer) -> Response:
        """Bulk revoke all sessions for multiple users"""
```